client/src/pages/ChangePasswordPage.jsx

```diff
@@ -0,0 +1,166 @@
+            <h1 className="title">Change password</h1>
+            <div className="field">
+              <label htmlFor="new-password" className="label">
+                New password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
+                  name="password"
+                  type="password"
+                  id="new-password"
+                  placeholder="*******"
+                  className={cn('input', {
+                    'is-danger': touched.password && errors.password,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-lock"></i>
+                </span>
+
+                {touched.password && errors.password && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.password && errors.password ? (
+                <p className="help is-danger">{errors.password}</p>
+              ) : (
+                <p className="help">At least 6 characters</p>
+              )}
+            </div>
+
+            <div className="field">
+              <label htmlFor="confirm-password" className="label">
+                Confirm password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
+                  name="confirmation"
+                  type="password"
+                  id="confirm-password"
+                  placeholder="*******"
+                  className={cn('input', {
+                    'is-danger': touched.confirmation && errors.confirmation,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-lock"></i>
+                </span>
+
+                {touched.confirmation && errors.confirmation && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.confirmation && errors.confirmation ? (
+                <p className="help is-danger">{errors.confirmation}</p>
+              ) : (
+                <p className="help">At least 6 characters</p>
+              )}
+            </div>
+
+            <div className="field">
+              <button
+                type="submit"
+                className={cn('button is-success has-text-weight-bold', {
+                  'is-loading': isSubmitting,
+                })}
+                disabled={
+                  isSubmitting || errors.password || errors.confirmation
```

Comment:
The `disabled` attribute for the submit button checks for `errors.password` and `errors.confirmation`. However, it should also consider whether the fields are `touched` to avoid disabling the button when the user hasn't interacted with the fields yet.